nn/conformer.py

```diff
@@ -79,9 +79,9 @@ class _ConformerConvSubsample(nn.Module):
   """
 
   def __init__(
-        self, filter_sizes: List[Tuple[int, int]], pool_sizes: Optional[List[Tuple[int, int]]],
-        channel_sizes: List[int], dropout: float = 0.3, activation: Callable[[nn.LayerRef], nn.LayerRef] = nn.relu,
-        padding: str = 'same'):
+        self, filter_sizes: List[Tuple[int, int]], channel_sizes: List[int],
+        pool_sizes: Optional[List[Tuple[int, int]]] = None, dropout: float = 0.3,
```

Comment:
I wonder about the `dropout` default. Is this really so universal? Maybe it should be explicit? Or 0 by default?

Reply:
Hmm yes this should be changed. ESPNet uses here the same dropout rate as the dropout layer applied just before the conformer blocks which is 0.1 by default.